src/api/lib/tasks/dev.rake

```diff
@@ -99,6 +99,61 @@ namespace :dev do
       end
     end
   end
+  namespace :testdata do
+    task create: :environment do
+      unless Rails.env.to_s == 'development'
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      Rake::Task['db:drop'].invoke
+      Rake::Task['db:create'].invoke
+      Rake::Task['db:setup'].invoke
+
+      iggy = create(:confirmed_user, login: 'Iggy')
+      admin = User.where(login: 'Admin').first
+      User.current = admin
+
+      interconnect = create(:project, name: 'openSUSE.org', remoteurl: 'https://api.opensuse.org/public')
+      tw_repository = create(:repository, name: 'snapshot', project: interconnect, remote_project_name: 'openSUSE:Factory')
+
+      home_admin = create(:project, name: admin.home_project_name)
```

Comment:
IIRC, every user factory has already a home project. So there should be a ` admin.home_project ` available

Reply:
Nope, we do this in many specs :/ But might make sense to introduce a factory like ``user_with_home``